client/extensions/zoninator/components/settings/zones-dashboard/index.jsx

```diff
@@ -4,58 +4,66 @@
 import React, { PropTypes } from 'react';
 import { connect } from 'react-redux';
 import { localize } from 'i18n-calypso';
-import { find, flowRight, get, noop } from 'lodash';
+import { find, flowRight, get, range } from 'lodash';
 
 /**
  * Internal dependencies
  */
-import Zone from './zone';
 import Button from 'components/button';
 import HeaderCake from 'components/header-cake';
 import SectionHeader from 'components/section-header';
 import sectionsModule from 'sections';
-import { getSelectedSiteSlug } from 'state/ui/selectors';
+import { getSelectedSiteId, getSelectedSiteSlug } from 'state/ui/selectors';
+import QueryZones from '../../data/query-zones';
+import ZoneItem from './zone-item';
+import ZonePlaceholder from './zone-placeholder';
+import { getZones, isFetchingZones } from '../../../state/zones/selectors';
 
-const zones = [
-	{ label: 'Foo', slug: 'foo', description: 'My first zone' },
-	{ label: 'Bar', slug: 'bar', description: 'Another zone' },
-	{ label: 'Baz', slug: 'baz', description: 'Another zone' },
-	{ label: 'Boo', slug: 'boo', description: 'Another zone' },
-];
+const placeholderCount = 16;
```

Comment:
Fine to use fewer, they don't have to extend all the way down. 4 or 5 should be fine.